src/porepy/fracs/fractures.py

```diff
@@ -619,8 +613,6 @@ def __init__(self, fractures=None, domain=None, verbose=0, tol=1e-8):
 
         Parameters:
             fractures (list of Fracture): Fractures that make up the network.
-            verbose (int, optional): Verbosity level. Defaults to 0.
-            tol (double, optional): Geometric tolerance. Defaults to 1e-4.
 
```

Comment:
No documentation on rest of parameters? What is the tolerance used for, which checks?